backend/internal/ee/rbac/enforcer/rbac_model.conf

```diff
@@ -11,4 +11,4 @@ g = _, _, _
 e = some(where (p.eft == allow))
 
 [matchers]
-m = g(r.sub, p.sub, r.dom) && r.dom == p.dom && keyMatch2(r.obj, p.obj) && keyMatch2(r.act, p.act)
+m = g(r.sub, p.sub, r.dom) && r.dom == p.dom && keyMatch(r.obj, p.obj) && keyMatch(r.act, p.act)
```

Comment:
after more inspection of the functions available, keyMatch is what we actually want.
https://casbin.org/docs/function/